spec/ruby/core/float/negative_spec.rb

```diff
@@ -0,0 +1,27 @@
+require_relative '../../spec_helper'
+
+describe "Float#negative?" do
+  describe "on positive numbers" do
+    it "returns false" do
+      0.1.negative?.should be_false
+    end
+  end
+
+  describe "on zero" do
+    it "returns false" do
+      0.0.negative?.should be_false
+    end
+  end
```

Comment:
Would be interesting to test on `-0.0` too

Reply:
Done... but I think there's a bug in MRI here https://bugs.ruby-lang.org/issues/17098.